app/viewModels/P5/ViewModelWithCustomsOffice.scala

```diff
@@ -22,21 +22,20 @@ import play.api.i18n.Messages
 trait ViewModelWithCustomsOffice {
 
   val prefix: String
-
-  val customsOffice: Either[String, CustomsOffice]
+  val customsOfficeId: String
+  val customsOffice: Option[CustomsOffice]
```

Comment:
Looks like we're always wrapping the customs office in an `Option` when passing into the view model apply method now? Let me know if that isn't the case. That would also negate the need of having the `customsOfficeId` in this trait.

Reply:
So the reason I have gone with this approach is because we have to match on the following `case None =>
        messages(s"$prefix.teleNotAvailAndOfficeNameNotAvail", customsOfficeId)` where we make use of the customOfficeId that was searched for. 
        The current changes to the service though suggest that we probably won't ever hit this case as we will be throwing the NoRefDataFoundException. Could we potentially clean up that case and hence not need the `val customsOfficeId: String` or need to wrap the CustomsOffice in an option?